include/inexor/vulkan-renderer/renderer.hpp

```diff
@@ -185,11 +172,15 @@ class VulkanRenderer {
 
     std::shared_ptr<DescriptorBundle> global_descriptor;
 
-    std::shared_ptr<MeshBuffer> octree_mesh;
-
     Camera game_camera;
 
     std::vector<Shader> shaders;
+    std::vector<Texture> textures;
+    std::vector<UniformBuffer> uniform_buffers;
+    std::vector<MeshBuffer> mesh_buffers;
+
+    // TODO(Hanni): Remove this with RAII refactoring of descriptors!
+    VkDescriptorImageInfo descriptor_image_info = {};
 
 public:
```

Comment:
Just a general hint. Group the `private`, `protected,  `public` (best in this order). There are also two `public` behind each other, one can be removed.

Reply:
done.